### DISCUSSION TITLE
FirebaseAuthentication error response

### DISCUSSION THREAD

**Discussion by jongbonga** [capawesome-team/capacitor-firebase#550] [Q&A] [answered]
Hey @robingenz 

when I try to linkWithEmailAndPassword with an already existing email, I notice the plugin returns a different message from the network tab

Network tab
```
{
  "error": {
    "code": 400,
    "message": "EMAIL_EXISTS",
    "errors": [
      {
        "message": "EMAIL_EXISTS",
        "domain": "global",
        "reason": "invalid"
      }
    ]
  }
}

```

console from my catch

![Screenshot 2024-01-30 at 15 12 27](https://github.com/capawesome-team/capacitor-firebase/assets/12111716/22043494-8565-4274-acb1-a14bf9e096aa)

is there a way to access the original error object?

**Comment by robingenz** [comment, marked as answer]
This error message comes directly from the Firebase JS SDK. So no, the plugin cannot do anything here.
